pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -415,23 +411,6 @@ func hasMemoryAndCPUInstUsage(info *cadvisorapiv2.ContainerInfo) bool {
 	return cstat.CpuInst.Usage.Total != 0 && cstat.Memory.RSS != 0
 }
 
-// isContainerTerminated returns true if the specified container info has
-// both zero CPU instantaneous usage and zero memory RSS usage, and
-// false otherwise.
-func isContainerTerminated(info *cadvisorapiv2.ContainerInfo) bool {
```

Comment:
it doesn't look like this bit considered whether network stats were being provided, and that was what we saw go missing